cli/command/formatter/container_test.go

```diff
@@ -357,7 +357,6 @@ func TestContainerContextWriteJSON(t *testing.T) {
 		t.Fatal(err)
 	}
 	for i, line := range strings.Split(strings.TrimSpace(out.String()), "\n") {
-		t.Logf("Output: line %d: %s", i, line)
```

Comment:
I have updated the PR: removal of Logf calls and addition of output details to assert calls.